code/_onclick/hud/swarmer.dm

```diff
@@ -68,11 +68,11 @@
 	var/obj/screen/using
 
 	using = new /obj/screen/swarmer/FabricateTrap()
-	using.screen_loc = ui_rhand
+	using.screen_loc = ui_hand_position(2)
```

Comment:
the true solution to things like this would be to give the huds that cheated (read: used these ui positions incorrectly) is to give them their own ui positions, but for now, it's fine.